app/models/feed.rb

```diff
@@ -15,4 +15,8 @@ def name
   def to_param
     token
   end
+
+  def expired?
+    (last_fetched || created_at || Time.zone.now) + 1.year < Time.current
```

Comment:
`Time.zone` is set by the controller, so we probably just want `Time.now` inside the models, so we can use them even outside the context of a web request.

Reply:
I lied, the linter is right. Time.zone is available outside controllers now.